package.json

```diff
@@ -77,6 +77,7 @@
 		"@babel/runtime-corejs3": "7.8.3",
 		"@babel/traverse": "7.8.3",
 		"@octokit/rest": "16.26.0",
+		"@octokit/webhooks": "7.1.0",
```

Comment:
Out of curiosity, do we include this dev dependency only for type checking?

Reply:
Yes, that's right.

Related to this guideline:

>For packages which do not distribute their own TypeScript types, you are welcomed to install and use the DefinitelyTyped community-maintained types definitions, if one exists.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#external-dependencies-1

Though in this case, it's a bit awkward in that it's neither from DefinitelyTyped, nor actually the type of the packages we _are_ using (`@octokit/rest`, `@actions/github`) or a dependent of those. 

The reason for this is that the _actual correct_ typings are neither very useful nor accurate:

https://github.com/actions/toolkit/blob/12f3011/packages/github/src/interfaces.ts#L15-L39

See also: https://github.com/actions/toolkit/issues/147

It's still a bit questionable whether we should do this, since technically there's not a strong guarantee that the payload matches the type from `@octokit/webhooks`, aside from the fact that they both purport to represent the structure of a webhook payload.